.babelrc

```diff
@@ -1,3 +0,0 @@
-{
-  "optional": ["runtime"]
```

Comment:
I actually think that we don't need that file at all anymore, since jscodeshift doesn't have a build process anymore since 327f985598784e1e91615d78237ece490995d165.